code/src/cljs/sixsq/nuvla/ui/filter_comp/views.cljs

```diff
@@ -322,6 +324,19 @@
                    (close-fn))}
      (@tr [:clear-filter])]))
 
+(reg-event-fx
+  ::call-filter-fn
+  [(inject-cofx :storage/all)]
+  (fn [{{:keys [current-route]} :db
+        storage :storage/all} [_ fn filter-query]]
+    (let [filter-storage-key (get-query-param current-route (keyword :filter-storage-key))
```

Comment:
keyword useless
`(keyword :filter-storage-key)`.

I don't understand why the filter component is in charge of deleting persisted query.